src/pkg/exporter/outputs/client.go

```diff
@@ -43,11 +43,11 @@ type Client struct {
 	CheckCert        bool
 	HeaderList       []Header
 	ContentType      string
-	Config           *types.Configuration
+	Config           *v1alpha1.ExportConfig
```

Comment:
Move the stuct definition to v1alpha1 because it will also be used by the scanners, not only exporter